data/db-schema/src/main/resources/net/corda/db/schema/config/migration/config-creation-v1.0.xml

```diff
@@ -311,5 +311,19 @@
                                  constraintName="FK_vnode_instance_cpi"
                                  baseTableSchemaName="${schema.name}"
                                  referencedTableSchemaName="${schema.name}"/>
+        <createTable tableName="cluster_certificate" schemaName="${schema.name}">
+            <column name="type" type="VARCHAR(255)">
```

Comment:
related to my comment above.. does this mean this value can either contain a certificate type or a holding identity value? Should we create separate columns for each piece of information?

Reply:
Also, if I understand correctly in case it's a vnode-level key/cert, we expect the user to specify the holding id short hash here and this means that we won't store the certificate type/usage in the DB at all?

Reply:
This is just for the cluster-level certificate (so, only type).  The virtual node certificates are in another table in the virtual node database.

Reply:
ah right, forgot about the other table. Thanks